protostar/commands/test/messages/passed_fuzz_test_case_result_message.py

```diff
@@ -0,0 +1,113 @@
+        )
+
+        info_items: list[str] = [
+            get_formatted_execution_time_human(
+                execution_time=self.passed_fuzz_test_case_result.execution_time,
+                log_color_provider=fmt,
+            )
+        ]
+
+        if self.passed_fuzz_test_case_result.fuzz_runs_count is not None:
+            info_items.append(
+                f"fuzz_runs={fmt.bold(self.passed_fuzz_test_case_result.fuzz_runs_count)}"
+            )
+
+        if self.passed_fuzz_test_case_result.execution_resources:
+            if (
+                self.passed_fuzz_test_case_result.execution_resources.estimated_gas
+                is not None
+            ):
+                info_items.append(
+                    f"gas={fmt.bold(self.passed_fuzz_test_case_result.execution_resources.estimated_gas)}"
+                )
+            if self.passed_fuzz_test_case_result.execution_resources.n_steps:
+                info_items.append(
+                    f"steps={fmt.bold(self.passed_fuzz_test_case_result.execution_resources.n_steps)}"
+                )
+            if self.passed_fuzz_test_case_result.execution_resources.n_memory_holes:
+                formatted_n_memory_holes = fmt.bold(
+                    self.passed_fuzz_test_case_result.execution_resources.n_memory_holes
+                )
+                info_items.append(f"memory_holes={formatted_n_memory_holes}")
+
+        if len(info_items) > 0:
+            info = ", ".join(info_items)
+            first_line_elements.append(fmt.colorize("GRAY", f"({info})"))
+
+        first_line = " ".join(first_line_elements)
+
+        second_line_elements: list[str] = []
+        if self.passed_fuzz_test_case_result.execution_resources:
+            for (
+                builtin_name,
+                builtin_count,
+            ) in (
+                self.passed_fuzz_test_case_result.execution_resources.builtin_name_to_count_map.items()
+            ):
+                if builtin_count:
+                    second_line_elements.append(
+                        fmt.colorize(
+                            "GRAY",
+                            f"{builtin_name}={fmt.bold(builtin_count)}",
+                        )
+                    )
+
+        stdout_elements = get_formatted_stdout(
+            captured_stdout=self.passed_fuzz_test_case_result.captured_stdout,
+            log_color_provider=fmt,
+        )
+
+        if len(second_line_elements) > 0 or len(stdout_elements) > 0:
+            second_line_elements.insert(0, "      ")
+            second_line = " ".join(second_line_elements)
+
+            # To maintain consistent spacing
+            to_join: list[str] = [first_line, second_line]
+            if stdout_elements:
+                to_join.append("".join(stdout_elements))
+            return "\n".join(to_join)
+
+        return first_line
+
+    def format_dict(self) -> dict:
+        result = {
+            "message_type": "test_case_result",
```

Comment:
I wouldn't remove the `message_type` field. I think we discussed this already in this PR (if necessary, I can find the discussion in resolved comments). The bottom line is:
- `message_type` distinguishes test outputs from each other, you can have `test_case_result`, `test_collector_result`, and `testing_summary`.
- `test_type` is a field only for `test_case_result` and it can be `broken_test_suite`, `failed_test_case`, etc.

I'm going to juz add `type=test`. I think having a few identifying fields like these is not expensive and can help a lot.

Reply:
[add type field](https://github.com/software-mansion/protostar/pull/1254/commits/dd26b46021fb5d27fb62177c4a7a492c067ea6c5)